### PR TITLE
[TEST] Add edge_test flags. @open sesame 8/26 09:34

### PR DESCRIPTION
**Pull request by gichan-jang**
If "unit_test" is disabled and "edge_test" is enabled, run query and edge test.

Path of the edge related test files are changed.
Before:
```
  tests
     |_ nnstreamer_edge
            |_ unittest_edge.cc
            |_ runTest.sh
     |_ nnstreamer_query
            |_ unittest_query.cc
            |_ runTest.sh
```
After
```
  tests
     |_ nnstreamer_edge
           |_ edge
               |_ unittest_edge.cc
               |_ runTest.sh
           |_ query
               |_ unittest_query.cc
               |_ runTest.sh
```
**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

File: packaging/nnstreamer.spec [hunk 1]
```diff
@@ -899,7 +899,15 @@ export NNSTREAMER_CONVERTERS=${NNSTREAMER_BUILD_ROOT_PATH}/ext/nnstreamer/tensor
     popd
 
 python3 tools/development/count_test_cases.py build tests/summary.txt
-%endif #if unit_test
+%else
+%if 0%{?edge_test}
+    bash %{test_script} tests/nnstreamer_edge
```
Comment (myungjoo) [inline packaging/nnstreamer.spec:904]: add tests/nnstreamer_query

Comment (gichan-jang) [reply packaging/nnstreamer.spec:905]: This is directory path.
`unittest_query` and `unittest_edge` execution files are in tests/nnstreaer_edge.
What I don't know is that ssat doesn't run `tests/nnstreamer_edge/query/runTesh.sh` and `tests/nnstreamer_edge/edge/runTesh.sh`. I think it should be run..

Comment (myungjoo) [reply packaging/nnstreamer.spec:905]: ok :)

**Comment by myungjoo** [comment]
You will need to wait for SR/Accept of nnstreamer-edge:HEAD to get all test cases passed.

**Review by myungjoo — APPROVED** [review]

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**